recipes/emsdk/all/conanfile.py

```diff
@@ -0,0 +1,145 @@
+import os
+from pathlib import Path
+
+from conan import ConanFile
+from conan.tools.build import cross_building
+from conan.tools.env import Environment, VirtualBuildEnv
+from conan.tools.files import chdir, copy, get
+from conan.tools.layout import basic_layout
+
+required_conan_version = ">=2.1"
+
+
+class EmSDKConan(ConanFile):
+    name = "emsdk"
+    description = "Emscripten SDK. Emscripten is an Open Source LLVM to JavaScript compiler"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/kripken/emscripten"
+    topics = ("emsdk", "emscripten", "sdk", "emcc", "em++", "nodejs")
+    license = "MIT"
+    package_type = "application"
+    settings = "os", "arch"
+    upload_policy = "skip"
+    build_policy = "missing"
```

Comment:
Do we now want this? It seems like being here would alleviate our concerns with sizes etc, but maybe this is now here for the cache things we discussed?